### PR TITLE
Add Docstring

### PR DESCRIPTION
**Pull request by StefanThoene**
remove get_cloud_volume (unused function)

### CODE REVIEW

**Comment by StefanThoene** [comment]
@pluAtAnsys cann you bhave a look on the Gascloud docstrings?

**Comment by beliaev-maksim** [comment]
@StefanThoene not quite a numpy style :)

https://numpydoc.readthedocs.io/en/latest/format.html

**Comment by pluAtAnsys** [comment]
@StefanThoene @beliaev-maksim style in numpy now

File: mustards_game/gas_cloud.py [hunk 1]
```diff
@@ -28,6 +28,17 @@ def __init__(self, position_x, position_y):
         self.gas_rect.move_ip(self.gas_position_x, self.gas_position_y)
 
     def set_color(self, level):
+        """
+        function to define the color of a gas pixel based on its level
```
Comment (beliaev-maksim) [inline mustards_game/gas_cloud.py:32]:
```suggestion
        """Function to define the color of a gas pixel based on its level.
        
```

File: mustards_game/gas_cloud.py [hunk 2]
```diff
@@ -28,6 +28,17 @@ def __init__(self, position_x, position_y):
...
+        Parameters
+        ----------
+        level: float
+            current gas level
```
Comment (beliaev-maksim) [inline mustards_game/gas_cloud.py:36]:
```suggestion
           Current gas level.
```

File: mustards_game/gas_cloud.py [hunk 3]
```diff
@@ -28,6 +28,17 @@ def __init__(self, position_x, position_y):
         self.gas_rect.move_ip(self.gas_position_x, self.gas_position_y)
 
     def set_color(self, level):
+        """
+        function to define the color of a gas pixel based on its level
+        Parameters
+        ----------
+        level: float
```
Comment (beliaev-maksim) [inline mustards_game/gas_cloud.py:35]:
```suggestion
        level : float
```

File: mustards_game/gas_cloud.py [hunk 4]
```diff
@@ -28,6 +28,17 @@ def __init__(self, position_x, position_y):
...
+            current gas level
+
+        Returns
+        -------
+
```
Comment (beliaev-maksim) [inline mustards_game/gas_cloud.py:40]:
```suggestion
```

File: mustards_game/gas_cloud.py [hunk 5]
```diff
@@ -38,7 +49,7 @@ def set_color(self, level):
 
 class GasCloud:
     """
-    this class describe the gas cloud separated by airplane
+    this class describe the gas cloud created by Ufo
     """
```
Comment (beliaev-maksim) [inline mustards_game/gas_cloud.py:53]: single line, start with capital, end with period.

File: mustards_game/gas_cloud.py [hunk 6]
```diff
@@ -53,15 +64,34 @@ def __init__(self):
...
-        :return: None
+        Parameters
+        ----------
+        position_x: int
+        position_y: int
+        altitude: int
+
+        Returns
+        -------
+
```
Comment (beliaev-maksim) [inline mustards_game/gas_cloud.py:75]:
```suggestion
        position_x : int
        position_y : int
        altitude : int

```

File: mustards_game/gas_cloud.py [hunk 7]
```diff
@@ -53,15 +64,34 @@ def __init__(self):
...
+        Parameters
+        ----------
+        screen: pygame.display
+            current displayed screen
+        position_x: int
+        position_y: int
+
+        Returns
+        -------
+
```
Comment (beliaev-maksim) [inline mustards_game/gas_cloud.py:93]:
```suggestion
        """Draw and distribute the Gas by pushing parts of its density to the direct neighbouring pixel.
        
        The gas degradation and drawing is limited to a 200x200 square around the given position.
        Parameters
        ----------
        screen : pygame.display
            Current displayed screen.
        position_x : int
        position_y : int

```

File: mustards_game/gas_cloud.py [hunk 8]
```diff
@@ -103,31 +133,35 @@ def degrade_gas(self, screen, position_x, position_y):
     def get_area_covered(self):
         """
         This function return area covered by GasCloud
-        :return: int
+        Returns: int
```
Comment (beliaev-maksim) [inline mustards_game/gas_cloud.py:136]:
```suggestion
        Returns
        -------
        area : int
```

**Review by beliaev-maksim — CHANGES_REQUESTED** [review]

**Comment by beliaev-maksim** [comment]
@pluAtAnsys @StefanThoene 
I reviewed some of the strings. Please correct the rest according to the guide

**Comment by pluAtAnsys** [comment]
@beliaev-maksim the pre-commit checking was failed. any idea?

**Comment by beliaev-maksim** [comment]
@pluAtAnsys 

1. pull the branch
2. fix other issues in other function
3. do a commit
4. let pre commit fix everything
5. push

**Pull request merged**